packages/dashboard/src/views/Holdings.js

```diff
@@ -100,17 +100,38 @@ const Holdings = ({
 
       const currAllData = historicQuotes['all']
       const currAllResults = currAllData.results[currAllData.results.length - 1]
-
-      // override current balance
+    
+      // Override cash value (buying power)
       smallStats[0] = {
         ...smallStats[0],
+        value: Math.round(selectedAccount.buying_power.amount * 100) / 100
+      }
+
+      // Override securities value
+
+      const positions = selectedAccount.positions
+    
+      let initialValue = 0
+      let securitesValue = positions.reduce(function (accumulator, currentValue) {
```

Comment:
If you wanna be even cleaner:
```
const securitiesValue = positions.reduce((acc, position) => acc + position.todays_earnings_baseline_value.amount, 0)
```

- ES6 arrow functions can (and _usually_ should be) used in place of regular functions
- if you never have to overwrite the value, always use `const`
- you can short form args just to compress things

No need to change rn tho, just for future learnings

Reply:
roger that